src/components/organizations/index.js

```diff
@@ -0,0 +1,9 @@
+import OrganizationList from "./OrganizationList";
+import OrganizationShow from "./OrganizationShow";
+import OrganizationIcon from "@material-ui/icons/Business";
+
+export default {
+  list: OrganizationList,
+  show: OrganizationShow,
+  icon: OrganizationIcon,
```

Comment:
missing `delete`

Reply:
There is no delete property available in the Resource component.